### PR TITLE
WebdriverIO sample code

### PR DESCRIPTION
**Pull request by dpgraham**

### CODE REVIEW

**Review by imurchie — COMMENTED** [review]
Looks good for tests.

File: sample-code/javascript-webdriverio/helpers/caps.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+var path = require('path');
```
[the remark this answers is not in the document]
Comment (dpgraham) [reply sample-code/javascript-webdriverio/helpers/caps.js]: I just used the style of JS that the wdio examples use. I'm fine with refactoring it to modern JS if you think that's how we should do it.

[the remark this answers is not in the document]
Comment (imurchie) [reply sample-code/javascript-webdriverio/helpers/caps.js]: I think it best to keep it vanilla, but make the minimum version of Node be 6 (like Appium) so there can be things like `const` and `let`. But still use `require` so there is no need to use fancy features or babel or anything.

**Comment by dpgraham** [comment]
Fixes pushed

**Review by imurchie — APPROVED** [review]

**Review by jlipps — APPROVED** [review]

**Pull request merged**